gamecenter/api/helpers/get.py

```diff
@@ -31,18 +35,23 @@ def construct_and_(args):
 
 
 def get_request_args(view_func):
-    """Make sure all arguments are passed to a view."""
+    """Make sure all URL and POST data arguments are passed to a view."""
+    @wraps(view_func)
     def get_args():
-        """Get the arguments from a request."""
+        """Get the URL and POST data arguments from a request."""
+        both = {k: v[0] for k,v in dict(request.args).iteritems()}
+        both.update(json.loads(request.data or "{}"))
         args = {}
-        compare_dates(start=request.args.get('start_date'), end=request.args.get('end_date'))
-        args['start_date'] = valid_start_date(request.args.get('start_date'))
-        args['page_size'] = valid_page_size(request.args.get('page_size'))
-        args['end_date'] = valid_end_date(request.args.get('end_date'))
-        args['user_id'] = valid_user_id(request.args.get('user_id'))
-        args['offset'] = valid_offset(request.args.get('offset'))
-        args['sort'] = valid_sort(request.args.get('sort'))
-        args['tag'] = valid_tag(request.args.get('tag'))
+        compare_dates(start=both.get('start_date'), end=both.get('end_date'))
+        args['start_date'] = valid_start_date(both.get('start_date'))
+        args['page_size'] = valid_page_size(both.get('page_size'))
+        args['end_date'] = valid_end_date(both.get('end_date'))
+        args['user_id'] = valid_user_id(both.get('user_id'))
+        args['offset'] = valid_offset(both.get('offset'))
+        args['sort'] = valid_sort(both.get('sort'))
+        args['tag'] = valid_tag(both.get('tag'))
+        args['filter_tag'] = valid_tag(both.get('filter_tag'))
```

Comment:
I don't think it's not easy, I think it's not *as easy* as it was before since they were organized by height